src/App.js

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
-import ToysContainer from './containers/ToysContainer'
+import ToysContainer from './containers/ToysContainer';
 import './App.css';
-import Navbar from "./components/Navbar";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Navbar from './components/Navbar';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Home from './pages';
 import About from './pages/about';
 import SignUp from './pages/signup';
 import SignIn from './pages/signin';
 
-class App extends React.Component {
-
-  render() {
-    return (
-      <Router>
+const App = () => {
+  return (
```

Comment:
Just remember that if you're just returning, you can clean things up a bit but using () instead of {} (then you don't need to explicitly include the `return`):

```
const App = () => (
  <Router>
      ...
  </Router>
);
```